locales/fr.json

```diff
@@ -0,0 +1,14 @@
+{
+  "home.description": "Générer très facilement divers liens de partage",
+  "home.description2": "Pas de pubs, Pas de suivi, Simple",
```

Comment:
Thank you for your contribution!
I don't understand French, but I checked with a machine translation.
Does `de suivi` mean `tracking` (e.g. Google Analytics)?
I was curious about this. I'm sorry if it's right!

Reply:
`suivi` means `tracking`. `de` is a french preposition

Reply:
`Traçage` may be a better word to use here. I will check it when I wake up

Reply:
Ok I checked the web and it seems that `suivi` is the word used (Like for Google Analytics they say `code de suivi Google Analytics`)

Reply:
OK, please mention when you're done! I'll merge it.